openvpn/fastestvpn/Czechia-UDP.ovpn

```diff
@@ -1,12 +1,12 @@
 client
-remote cz-pr-ovudp-01.jumptoserver.com 4443
+remote cz.jumptoserver.com 4443
 proto udp
 comp-lzo
 persist-key
 persist-tun
 dev tun
 auth SHA256
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass
```

Comment:
yeah..these files all need to be fixed, the above line should not be removed

Reply:
You still haven’t fixed the config. The above line is edited to remove the path to /config.. that is wrong.
Without that line it won’t work

Reply:
Sorry man, I looked at the proto part, because I thought the > was where you wanted me to focus.

I'm absolutely brand new to this, and haven't contributed previously, so I'd appreciate your patience with this.

Reply:
There is a script in the openvpn folder, 
modify-openvpn-config.sh That should check and update all files in one go ;)

Reply:
Alright, I think I did it right this time! Thank you!